src/__tests__/__snapshots__/fjs_babel-test.js.snap

```diff
@@ -457,6 +459,20 @@ exports[`test if-line-comment.js 1`] = `
 "
 `;
 
+exports[`test jsx.js 1`] = `
+"var single = \'quotes\';
+var outnumber = \'double\';
+var moreSingleQuotesThanDouble = \'!\';
+
+React.createClass({ render() {
```

Comment:
Yeah breaking scopes if any child scope breaks is something I need to add into the rules. There are quite a few other places where this makes sense to do.